docs/en/registry-catalogue.rst

```diff
@@ -0,0 +1,338 @@
+
+    - Download the basic version of the Digital Credentials Catalogue (compact, without localisations) using the ``.well-known/credential-catalogue`` endpoint.
+    - Determine the User's preferred language.
+    - Download only the necessary localisation bundles.
+    - Dynamically merge localised content with the Digital Credentials Catalogue structure.
+
+  A non-normative example of a localisation bundle output is given below:
+
+    .. code-block:: 
+
+      {
+        "driving_license.name": "Patente di Guida",
+        "driving_license.description": "Patente di guida ufficiale valida in Italia e nell'UE",
+        "purpose.driving_authorization.name": "Abilitazione alla guida",
+        "purpose.driving_authorization.description": "Verifica di Abilitazione alla guida",
+        "claims.given_name.name": "Nome",
+        ...
+      }
+
+  Localization bundles MUST be available at the URI specified in the **localization_info.bundles_base_uri** claim of the Digital Credentials Catalogue. Each locale bundle MUST be accessible following the naming pattern **{locale_code}.json**, where **{locale_code}** is replaced with the corresponding locale code from the **available_locales** array.
+
+  A non-normative example of the Italian localization URI for the mDL bundle would be **https://trust-registry.eid-wallet.example.it/.well-known/l10n/mdl/it.json**.
+
+  Entities SHOULD verify the integrity of downloaded localization bundles using the digest method and values specified in the **localization_info.integrity** claim. This ensures that the localization data has not been tampered with during transmission.
+
+
+Claims Taxonomy
+^^^^^^^^^^^^^^^^
+
+The Digital Credential Catalogue, MUST include also a reference URI to Claim Taxonomy providing, in a single resource, the semantic information of all registered and available claims within the IT-Wallet ecosystem. It MUST be Credential format neutral and has the aim of facilitating Digital Credentials integrations in the IT-Wallet technical solutions.
+
+A non-normative example of the Claim Taxonomy is given below.
+
+.. literalinclude:: ../../examples/catalogue-claims-taxonomy.json
+  :language: JSON
+
+
+Digital Credentials Catalogue Endpoint
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The Digital Credentials Catalogue Endpoint MUST be a well-known HTTPS URI [:rfc:`8615`] that provides public access to information related to the Credentials available in IT-Wallet.
+
+Digital Credentials Catalogue Request
+---------------------------------------
+
+The Digital Credentials Catalogue Request MUST be an HTTP GET using the application/jose media type as in the following non-normative example.
+
+.. code-block:: http
+
+    GET /.well-known/credential-catalogue HTTP/1.1
+    Host: www.trust-registry.eid-wallet.example.it
+    Content-Type: application/jose
```

Comment:
if application/json the content must be in json plain text

Reply:
We set content-Type to application/jose

What do you mean with your comment?

Reply:
This https://github.com/italia/eid-wallet-it-docs/issues/641

for v1.1.0